runners/samza/src/main/java/org/apache/beam/runners/samza/translation/TranslationContext.java

```diff
@@ -159,12 +182,29 @@ public MessageStream<OpMessage<String>> getDummyStream() {
   }
 
   public <OutT> MessageStream<OpMessage<OutT>> getMessageStream(PValue pvalue) {
+    return getMessageStream(pvalue, true);
+  }
+
+  public <OutT> MessageStream<OpMessage<OutT>> getMessageStream(
+      PValue pvalue, boolean enableTransformMetric) {
     @SuppressWarnings("unchecked")
     final MessageStream<OpMessage<OutT>> stream =
         (MessageStream<OpMessage<OutT>>) messsageStreams.get(pvalue);
     if (stream == null) {
       throw new IllegalArgumentException("No stream registered for pvalue: " + pvalue);
     }
+
+    // add a step to attach InputMetricOp if registered for Op Stream
+    final Config overrideConfig = new MapConfig(getPipelineOptions().getConfigOverride());
```

Comment:
Use pipeline options instead of samza config.

Reply:
done